cmd/tink-worker/client/client.go

```diff
@@ -0,0 +1,59 @@
+package client
```

Comment:
this is specifically a tink client, no? we might want to use `tink` as the package name. That would remove the need to have tink in any of the function names. Then we could have functions like `tink.TLSCreds`, `tink.ServerCreds`, `tink.Conn`, etc.

Reply:
> this is specifically a tink client, no? we might want to use `tink` as the package name. That would remove the need to have tink in any of the function names. Then we could have functions like `tink.TLSCreds`, `tink.ServerCreds`, `tink.Conn`, etc.

I thought tink-worker package is called tink ?

Reply:
Per your suggestion, I've moved this to `tink-worker/client/tink.go`